ci/vm/entrypoint.sh

```diff
@@ -28,6 +28,7 @@ install_packages() {
 xpmem_build() {
   git clone -q https://github.com/openucx/xpmem.git
   cd xpmem
+  git checkout "$BUILD_SOURCEVERSION"
```

Comment:
Maybe better to use AZP checkout functionality and pass the folder to the VM? Currently it looks like you checkout twice.

Reply:
I do checkout twice. Mapping a volume from VM would be a valid alternative. From my experiments, it takes longer to bring up VM each time, this is due to additional components being installed and started.
Pulling the 3Mb repo again is a faster solution for this case.